selenium_tests/course_info/page_objects/course_people_page_object.py

```diff
@@ -25,7 +25,7 @@ def DELETE_USER_ICON (cls, sis_user_id):
     def TD_TEXT_XPATH(cls, search_text):
         """ returns a locator for a table cell element in the people table;
         search_text should be user's name, user_id, etc """
-        return By.XPATH, '//td[contains(text(), "{}")]'.format(search_text)
+        return By.XPATH, '//td[text()="{}")]'.format(search_text)
```

Comment:
This should match exact text, which is better than `contains()`, but it does depend on your use case -- presumably the other tests that are making use of this PO will still work (that is, their use case is also to search for an exact match).